build.gradle

```diff
@@ -129,8 +129,6 @@ subprojects {
 }
 
 ext {
-    neo4jVersionEffective = project.hasProperty("neo4jVersionOverride") ?
-            project.getProperty("neo4jVersionOverride")
-            : (System.env.CI != null) ? "5.2.0-dev" : "5.2.0-SNAPSHOT"
+    neo4jVersionEffective = project.hasProperty("neo4jVersionOverride") ? project.getProperty("neo4jVersionOverride") : "5.2.0"
```

Comment:
This looks fine to me, but can you explain why this changes from dev/snapshot now, I thought 5.2 is to be released Friday

Reply:
Answered privately